### DISCUSSION TITLE
TVM

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#728] [Q&A]
**Channel Name:** TVM 
**Channel ID:** TVM.mt
**Country:** MALTA
**Language:** xxx
**Website:** https://www.tvm.com.mt/mt/tvmi/live/
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by eltoncomp** [comment]
TVM Malta is already in the playlist, however the stream doesn't open, so I suggest that the label "channel request" be changed to "broken stream".

**Comment by MapGuy11** [comment]
https://live01.delivery.twentythree.com/live/amlst:fqo3sk6nph2iqa93tfaaj92ceg:71154069-1278e729306f170d3483.mp4/playlist.m3u8

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/mt_smash_plus.m3u#L46-L47
